azext_iot/central/models/v1_1_preview/destination.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+class Destination:
+    def __init__(self, destination: dict):
+        self.id = destination.get("id")
```

Comment:
Is this auto generated? I would use this as a starting point for what parameters to add to help break down content (regarding comments about export destination create and update)

Reply:
content break down during creation